plotly/plotly/plotly.py

```diff
@@ -1488,9 +1488,10 @@ def get_grid(grid_url, raw=False):
     # check if request is ok
     if not r_meta.ok:
         raise exceptions.PlotlyError(
-            "Whoops, you got a 404 response. Your grid url {} may be "
+            "Whoops, you got a 404 response. {} may be "
```

Comment:
@theengineear decided to go with a custom message that states it's a `404` because this http message could mean at least 1) grid is private 2) it's not a grid but a plot, etc.